packages/react/src/components/OverflowMenu/next/OverflowMenu-test.js

```diff
@@ -99,7 +97,11 @@ describe('OverflowMenu (enable-v12-overflowmenu)', () => {
     );
     await userEvent.type(screen.getByRole('button'), 'enter');
     expect(screen.getByRole('button')).toHaveAttribute('aria-expanded', 'true');
-    await userEvent.click(document.body);
+
+    await act(async () => {
+      await userEvent.click(document.body);
+    });
```

Comment:
Is this for pulling focus out of the menu? I didn't think calls to `userEvent` needed to be wrapped in `act()`.